components/Workspaces/WorkspaceHeader.tsx

```diff
@@ -47,6 +49,17 @@ export const WorkspaceHeader = ({ workspace, children }: WorkspaceHeaderProps) =
       </h1>
       <div className="flex gap-2 justify-end">
```

Comment:
refactor (blocking): On smaller screens, the feature is not responsive. It causes other buttons to go off screen and will cause the layout for any page using the WorkspacesHeader to break.

![CleanShot 2024-08-29 at 08 22 30](https://github.com/user-attachments/assets/8762e1af-0b95-474a-a431-3fbe9306d8d7)

Adding flex wrap should sort it out. 👇

![CleanShot 2024-08-29 at 08 27 52](https://github.com/user-attachments/assets/6183cf81-560a-45c2-b937-1a7924331b15)

```suggestion
      <div className="flex gap-2 justify-end flex-wrap">
```